lea/clients/duckdb.py

```diff
@@ -130,7 +130,10 @@ def _table_reference_to_view_key(self, table_reference: str) -> tuple[str]:
         ('schema', 'subschema', 'table')
 
         """
-        database, leftover = table_reference.split(".", 1)
+        try:
+            database, leftover = table_reference.split(".", 1)
+        except ValueError:
+            raise Exception("Specify the full path to the table, e.g. 'schema.table'")
```

Comment:
I think this a useful thing to catch. However, I would add a bit more context. Maybe include the `table_reference` value to make the error message more informative?